types/node/http2.d.ts

```diff
@@ -583,7 +583,7 @@ declare module "http2" {
     }
 
     export class Http2ServerRequest extends stream.Readable {
-        private constructor();
+        constructor();
```

Comment:
The arguments are missing, see https://github.com/nodejs/node/blob/0481a7f3b883862cadf963314d23490adc6656a4/lib/internal/http2/compat.js#L274
As long as they were private it was not needed to specify the arguments as it was anyway not possible to call it but if you extend it you have to pass the correct args here.

Reply:
This is now done.